Source/platform/ios/paths.m

```diff
@@ -0,0 +1,30 @@
+#import <Foundation/Foundation.h>
+
+#include "paths.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const char *IOSGetPrefPath()
+{
+	@autoreleasepool {
+		NSArray *array = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+
+		if ([array count] > 0) {
+			NSString *str = [array objectAtIndex:0];
+			str = [str stringByAppendingString:@"/"];
+			const char *base = [str fileSystemRepresentation];
+
+			char *copy = malloc(strlen(base) + 1);
```

Comment:
Doesn't this leak memory since it is never freed?

Reply:
It does. I'll take to free it after it is used.